### DISCUSSION TITLE
有没有 查询得到 数据库实体表的子类的方法？

### DISCUSSION THREAD

**Discussion by sqlite-me** [dotnetcore/FreeSql#839] [Q&A]
现有A, B两张表和对应的实体，现在手动创建了一个实体C（要返回给前端的）, C继承自A，并增加了一些字段 是来自B的。
使用A Inner join B 查询，在ToList的时候，不想一个个地写来自A的字段，有没有什么方法实现。（类似ToDatatable(new{a,Property1=b.Field1,Property2=b.Field12})的写没）

**Comment by 2881099** [comment]
```c#
.ToList<C>();
```
